src/components/Page/Toolbar/Toolbar.jsx

```diff
@@ -67,26 +86,33 @@ export class Toolbar extends Component {
 
     return (
       <div className="toolbar">
-        <div ref={this.brushTopToolbarRef} className={`top-toolbar z-depth-1 valign-wrapper ${boardState.tool === 'brush' ? '' : 'hide'}`}>
+        <div ref={this.brushTopToolbarRef} className={`top-toolbar z-depth-1 valign-wrapper ${boardState.tool === 'brush' || 'rainbow_brush' ? '' : 'hide'}`}>
```

Comment:
Shouldn't this be `boardState.tool === 'brush' || boardState.tool === 'rainbow_brush'`?

Reply:
oops 😝  just missed this will correct it wait